src/wolfsftp.c

```diff
@@ -4685,6 +4686,9 @@ int SFTP_GetAttributes_Handle(WOLFSSH* ssh, byte* handle, int handleSz,
 static int SFTP_GetAttributes(void* fs, const char* fileName,
         WS_SFTP_FILEATRB* atr, byte noFollow, void* heap)
 {
+    (void) fs;
```

Comment:
Please move these below the variable declaration

Reply:
(aside) good call, done!